src/pages/summerGardening.js

```diff
@@ -0,0 +1,30 @@
+import  ReactPlayer from "react-player";
+import {videoData} from "../data/videodataList";
+import "../displaycomponents/category/category.css";
+import LikeButton from "../displaycomponents/likeButton/likefunctionality.js";
+import WatchLaterButton from "../displaycomponents/watchLaterButton/watchLater";
+function SummerVideoList(){
+
+    return(<div>
+        <h3>Summer Special Gardening Videos</h3>
+        <div className="video-list-display">
+        {
+                videoData.map((item)=>{
+                    if(item.category==="summer"){
+                        return (<div className="video-style">
```

Comment:
I guess you can create a component for the same as I have observed similar code in different file, just a suggestion